mypy/nodes.py

```diff
@@ -1732,13 +1732,13 @@ class TypeAliasExpr(Expression):
     """Type alias expression (rvalue)."""
 
     type = None  # type: mypy.types.Type
-    line = None  # type: int
+    fback = None  # type: mypy.types.Type
```

Comment:
Would be nice to have a comment that clarifies what "fback" means. (Apparently fallback? Why not spell it in full?)